gradio/blocks.py

```diff
@@ -631,9 +629,12 @@ def get_block_instance(id: int) -> Block:
             # URL of C, not B. The else clause below handles this case.
             if block_config["props"].get("proxy_url") is None:
                 block_config["props"]["proxy_url"] = f"{proxy_url}/"
+            postprocessed_value = block_config["props"].pop("value", None)
```

Comment:
Previously, we were using `block._skip_init_processing` to achieve the same thing. However, the problem is that `_skip_init_processing` was assigned after the component was constructed, so it was having no effect during the construction process, which is where its needed. I've removed the `_skip_init_processing` parameter and just directly applied this logic